src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml.cs

```diff
@@ -65,17 +65,16 @@ public DetailsLayoutBrowser() : base()
 
 		protected override void ItemManipulationModel_ScrollIntoViewInvoked(object? sender, ListedItem e)
 		{
-			FileList.ScrollIntoView(e);
-			ContentScroller?.ChangeView(null, FileList.Items.IndexOf(e) * Convert.ToInt32(Application.Current.Resources["ListItemHeight"]), null, true); // Scroll to index * item height
+			(FileList.ContainerFromItem(e) as ListViewItem)?.StartBringIntoView();
```

Comment:
This should work with or without grouping, but sometimes it scrolls as expected and sometimes it does not scroll at all.
I have no idea how to make it work all the time.